apps/docs/src/guide/contracts/minted-token-asset-id.md

```diff
@@ -0,0 +1,20 @@
+# Minted Token Asset ID
+
+The asset ID of a token on the Fuel network is determined by two factors:
+
+- A sub-identifier (Sub ID).
+- The ID of the contract that minted the token.
```

Comment:
```suggestion
- The ID of the contract that minted the token,
- A sub-identifier (Sub ID)
```
The hash is defined as `sha256(CONTRACT_ID ++ SUB_IDENTIFIER)` so it's better to order these bullet points the same way.